FiveCalls/FiveCalls/EditLocationViewController.swift

```diff
@@ -37,6 +38,7 @@ class EditLocationViewController : UIViewController, CLLocationManagerDelegate {
         
         if case .address? = UserLocation.current.locationType {
```

Comment:
I think before we enable the button, we should also validate `UserLocation.current.locationValue` has anything. My simulator has the legacy location data so it returns like so:

```type: address value:  | display: My Location```

Which causes the button to be enabled with nothing in it. 

![simulator screen shot apr 17 2017 8 47 48 pm](https://cloud.githubusercontent.com/assets/45654/25111207/218fe1d4-23af-11e7-9f3b-15dcd6201992.png)

Maybe call `addressTextFieldChanged` so that we're changing the `isEnabled` in one place?